js-library/src/contracts/contracts-arbitrum_goerli.env

```diff
@@ -0,0 +1,5 @@
+ARBITRUM_GOERLI_ENS_REGISTRY_ADDRESS=0x948456fc539cd88b7c9b5e7ca6ab6d2fc74a7060
```

Comment:
is it ok that `ARBITRUM_GOERLI_ENS_REGISTRY_ADDRESS` is equal to `OPTIMISM_GOERLI_ENS_REGISTRY_ADDRESS`?

Reply:
Good catch! ENS and Ratings addresses were permuted. That's because first time when I executed the deployment script it skipped ENS because there were no recent changes. Now the addresses are good. You can check it in etherscan:
https://goerli.arbiscan.io/address/0x6a8494eC853F00C78c18C3850f16D4A36B4c802A
https://goerli-optimism.etherscan.io/address/0x6a8494ec853f00c78c18c3850f16d4a36b4c802a

Reply:
I redeployed ENS contracts to arbitrum again, just in case. But still when I try to login with blossom it's returning `Transaction reverted`, although this time certainly deployment was correct.